### PR TITLE
added the solution

### PR DESCRIPTION
**Pull request by vamsikrishna73**

### CODE REVIEW

File: python_vamsikrishna73.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+n = int(input())
```
Comment (prasannakumar414) [inline python_vamsikrishna73.py]: change variable name for n and a

**Review by prasannakumar414 — COMMENTED** [review]
## increase readability
- The variable names must be improved.
-  As here the time of submission is not important it is important that your code should be understood by others not only by you.
-  use names like 'number' , 'binaryNumber'.
-  Learn about casing example : CamelCase etc..

File: python_vamsikrishna73.py (hunk 1 above)
Comment (prasannakumar414) [reply python_vamsikrishna73.py]: also learn casing for variable names.

Comment (vamsikrishna73) [reply python_vamsikrishna73.py]: ok

File: python_vamsikrishna73.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+number = int(input("enter an integer"))
+binaryNumber = ''
+while n>0:
```
Comment (prasannakumar414) [inline python_vamsikrishna73.py]: wrong variable

File: python_vamsikrishna73.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    binaryNumber += str(n%2)
```
Comment (prasannakumar414) [inline python_vamsikrishna73.py]: there is no variable with name n

File: python_vamsikrishna73.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+number = int(input("enter an integer"))
```
Comment (prasannakumar414) [inline python_vamsikrishna73.py:1]: \n after output

**Review by prasannakumar414 — APPROVED** [review]

**Pull request merged**